hudi-flink/src/test/resources/log4j-surefire.properties

```diff
@@ -15,7 +15,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ###
-log4j.rootLogger=WARN, CONSOLE
```

Comment:
Do we need to change the log level in this PR?

Reply:
I think it's better to change the log level. The `StreamWriteITCase#testWriteToHoodie` only throw following exception stacktrace if we keep `WARN` level.

```
org.opentest4j.AssertionFailedError: 
Expected :true
Actual   :false
<Click to see difference>
	at org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:55)
	at org.junit.jupiter.api.AssertTrue.assertTrue(AssertTrue.java:40)
	at org.junit.jupiter.api.AssertTrue.assertTrue(AssertTrue.java:35)
	at org.junit.jupiter.api.Assertions.assertTrue(Assertions.java:162)
	at org.apache.hudi.operator.utils.TestData.lambda$checkWrittenFullData$5(TestData.java:198)
```

<br />

While in flink project, `ArrayIndexOutOfBoundsException` is `INFO` level.
```
7919 [flink-akka.actor.default-dispatcher-2] INFO  org.apache.flink.runtime.jobmaster.JobMaster  - Trying to recover from a global failure.
java.lang.ArrayIndexOutOfBoundsException: 3
	at org.apache.hudi.operator.StreamWriteOperatorCoordinator.handleEventFromOperator(StreamWriteOperatorCoordinator.java:181)
	at org.apache.flink.runtime.operators.coordination.OperatorCoordinatorHolder.handleEventFromOperator(OperatorCoordinatorHolder.java:191)
	at org.apache.flink.runtime.scheduler.SchedulerBase.deliverOperatorEventToCoordinator(SchedulerBase.java:952)
	at org.apache.flink.runtime.jobmaster.JobMaster.sendOperatorEventToCoordinator(JobMaster.java:473)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.flink.runtime.rpc.akka.AkkaRpcActor.handleRpcInvocation(AkkaRpcActor.java:284)
	at org.apache.flink.runtime.rpc.akka.AkkaRpcActor.handleRpcMessage(AkkaRpcActor.java:199)
	at org.apache.flink.runtime.rpc.akka.FencedAkkaRpcActor.handleRpcMessage(FencedAkkaRpcActor.java:74)
	at org.apache.flink.runtime.rpc.akka.AkkaRpcActor.handleMessage(AkkaRpcActor.java:152)
```

Reply:
By the way, it's better to change the log level in unit test cases from my side, `INFO` level can let project output more helpful logs.

Reply:
fine, sounds reasonable. I just want to know the reason.